packages/lib/src/components/internal/IbanInput/IbanInput.tsx

```diff
@@ -32,6 +33,18 @@ interface IbanInputState {
     cursor: number;
 }
 
+const ibanHolderNameErrorObj: GenericError = {
+    isValid: false,
+    errorMessage: 'ach.accountHolderNameField.invalid', // TODO create bespoke translation key
+    error: 'ach.accountHolderNameField.invalid'
+};
+
+const ibanErrorObj: GenericError = {
+    isValid: false,
+    errorMessage: 'sepaDirectDebit.ibanField.invalid',
```

Comment:
It looks weird to have both `error` and `errorMessage` containing the same values.  I'd expect the`errorMessage` to already has the error message and not the i18n error key value. 

Can't error message be derived from the `error` property?

Reply:
I agree. And I got rid of it. Then I found that the visual errors stopped showing up. It seems that the visual errors rely on the `error` prop and the screenreader messages derive from the `errorMessage` prop.
This is something that can hopefully be fixed in a future iteration.